packages/core/src/api/guild.ts

```diff
@@ -164,6 +166,26 @@ export class GuildsAPI {
 	public async delete(guildId: Snowflake, { signal, reason }: Pick<RequestData, 'reason' | 'signal'> = {}) {
 		await this.rest.delete(Routes.guild(guildId), { reason, signal });
 	}
+	
+	/**
+	 * Add user to the guild
+         *
+         * @see {@link https://discord.com/developers/docs/resources/guild#add-guild-member}
+         * @param guildId - Id of the guild that will be added to the user's guilds
+	 * @param userId - Id of the user that will be added to guild
+         * @param body - The options for adding user to guild
+	 */
+	public async addMember(
```

Comment:
Doesnt it need a token?

Reply:
yes, body parameter has `access_token` property

Reply:
Okey then never mind